Dictionary.java

```diff
@@ -0,0 +1,49 @@
+import java.util.LinkedList;
+
+public class Dictionary {
+	private LinkedList<String> words;

```

Comment:
Could this be a different data structure than LinkedList (better for the complexity of performed operations)?